libs/scraper/internal/commitSimilar.ts

```
@@ -1,7 +1,7 @@
 import { execSync } from 'child_process';
 import { readFileSync, existsSync } from 'fs';
 import { join } from 'path';
-import type { ElementData } from '../src/types';
+import type { ElementData } from '@thecointech/scraper-types';
```

ESLint

[error] 'ElementData' is defined but never used.

(no-unused-vars)